datamegh/api/http.py

```diff
@@ -1 +1,38 @@
-""" HTTP Connection API. """
+""" Basic HTTP verbs for your API connection. """
+
+import requests
+
+
+def get(url, params=None, **kwargs):
```

Comment:
Doc strings. Every function requires doc strings. 
```suggestion
def get(url, params=None, **kwargs):
    """ Sends an HTTP GET request. """
```

Reply:
Same for every other functions.